packages/search/addon/components/navi-search-result/definition.ts

```diff
@@ -5,19 +5,47 @@
  * A component that displays results for definitions
  */
 
-import Component from '@glimmer/component';
+import NaviBaseSearchResultComponent from './base';
+import { tracked } from '@glimmer/tracking';
 import { inject as service } from '@ember/service';
 
-export default class NaviDefinitionSearchResultComponent extends Component {
+/**
+ * @constant NUM_TOP
+ */
+const NUM_TOP: number = 2;
+
+export default class NaviDefinitionSearchResultComponent extends NaviBaseSearchResultComponent {
   /**
    * @property {Ember.Service} metadataService
    */
   @service('bard-metadata') metadataService!: TODO;
 
+  /**
+   * @property {Boolean} showTop
+   */
+  @tracked showTop: Boolean = true;
+
   /**
    * @property {Boolean} hasMultipleDataSources
    */
-  get hasMultipleDataSources() {
+  get hasMultipleDataSources(): Boolean {
     return this.metadataService.loadedDataSources.length > 1;
   }
+
+  /**
+   * @property {Boolean} hasMoreResults
+   */
+  get hasMoreResults(): Boolean {
+    return this.args?.data.length > NUM_TOP;
+  }
+
+  /**
+   * @property {Array} results
+   */
+  get results(): Array<Object> {
```

Comment:
Do you see other `ResultComponent`s using show more/show less?

Reply:
I'm opening a PR for asset too to have show more/less functionality

Reply:
Yes, after testing in Spotlight it feels cluttered having too many results being shown but sometimes more results are relevant. I'm making the Asset component like that too now and I see the rest of the components to work like that.

Reply:
Would it be simpler to set a max height on the result component and set overflow-y to scroll?

Reply:
I already made the search panel scrollable, so it would be weird to have two levels of scrolling (I merged the scrollable search results change while developing this, I should update the gif)

Reply:
(updated the gif)

Reply:
We have two levels of scrolling in dashboards already. I think it would be much simpler to use that approach.

Reply:
Hmm, having a double scroll inside a dropdown feels a bit clunky i think. Though it's easier to implement a double scroll, it might be better to just stick to clicks as the main action within the dropdown.  I hate that users would even have to scroll at all in a search dropdown. If the search is smart enough, hopefully they never have to, but this could give us a way out until search gets smarter.

Reply:
I agree about needing to scroll in a search dropdown. Maybe a better long-term solution would be to have the dropdown return only a few results that match best and a dedicated search route that shows all of the expanded results. This is very similar to how google drive handles their search and I think it works well for our use-case too.

Reply:
> Do you see other ResultComponents using show more/show less?

This was a question to see if we should move some of this logic to the base class.